packages/components-angular/projects/components/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@swisspost/design-system-components-angular",
+  "version": "1.0.0",
+  "description": "Swiss Post Design System - Angular Wrapper Components",
+  "author": "Swiss Post <oss@post.ch>",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/swisspost/design-system.git"
+  },
+  "homepage": "https://swisspost-web-frontend.netlify.app",
+  "bugs": {
+    "url": "https://github.com/swisspost/design-system/issues"
+  },
+  "private": true,
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "restricted",
```

Comment:
Is the private flag here intentional? With `access: restricted` and `private: true`, the components don't get published.